### DISCUSSION TITLE
Have you planned on eventually including this version in the Edge Add-ons Store?

### DISCUSSION THREAD

**Discussion by SecularSteve** [uBlockOrigin/uBOL-home#4] [Q&A] [answered]
Or is it planned for only after the testing is finished?

**Comment by gorhill** [comment, marked as answer]
I don't use Windows and no developer account with Microsoft, at this point trying to publish something in the Microsoft store would just add another burden I can't take on.

**Reply by SecularSteve** [reply]
Thank you for the answer.
And there's no need to force yourself into doing things you don't like. I was just curious since the original UB is already included on there:
https://microsoftedge.microsoft.com/addons/detail/ublock-origin/odfafepnkmbhccpbejgmiehpchacaeak
Have a nice day, good luck, and with kind regards,

**Comment by ghost** [comment]
@nicole-ashley will you include mv3 version in your script?

**Reply by nicole-ashley** [reply]
It would be quite easy for me to add. However I didn't automatically do it because I wanted to have a conversation about if @gorhill wanted to do it under his own name this time, considering how difficult that is to chance once there's a big user base. 

I know time and resources are scarce, but I'd be happy to either provide the automation scripts (which can be run on Azure at little to no cost, probably cents per month), or I'd even be happy to host the automation myself if @gorhill provided me the API keys after setting up his publisher account.

**Reply by gorhill** [reply]
I will see what registering with Microsoft involves when I get the time and provide feedback on this.

**Comment by gorhill** [comment]
https://microsoftedge.microsoft.com/addons/detail/ublock-origin-lite/cimighlppcgcoapaliogpjjdehbnofhn
